website/docs/developer/development-guide/code/linting.md

```diff
@@ -0,0 +1,13 @@
+---
+id: linting
+title: Running PyLint Against Your Code
+description: Instructions on running PyLint.
+---
+
+## Running PyLint
+1. Navigate to the task's base folder.
+1. Activate the virtual environment.
```

Comment:
May want to add a step or reference a link to setting up the virtual environment.

Reply:
I was hoping they'd read the intro first, but you're right. Should I put a 'tip' on each affected page then?

Reply:
Done